src/imageproxy.rs

```diff
@@ -119,6 +119,25 @@ pub struct ImageProxyConfig {
     pub authfile: Option<String>,
     /// If set, disable TLS verification.  Equivalent to `skopeo --tls-verify=false`.
     pub insecure_skip_tls_verification: Option<bool>,
+    /// Provide a configured [`std::process::Command`] instance.
+    ///
+    /// This allows configuring aspects of the resulting child `skopeo` process.
+    /// The intention of this hook is to allow the caller to use e.g.
+    /// `systemd-run` or equivalent containerization tools.  For example you
+    /// can set up a command whose arguments are `systemd-run -Pq -p DynamicUser=yes -- skopeo`.
+    /// You can also set up arbitrary aspects of the child via e.g.
+    /// [`current_dir`] [`pre_exec`].
+    ///
+    /// [`current_dir`]: https://doc.rust-lang.org/std/process/struct.Command.html#method.current_dir
+    /// [`pre_exec`]: https://doc.rust-lang.org/std/os/unix/process/trait.CommandExt.html#tymethod.pre_exec
+    ///
+    /// The default is to wrap via util-linux `setpriv --pdeathsig SIGTERM -- skopeo`,
+    /// which on Linux binds the lifecycle of the child process to the parent.
+    ///
+    /// Note that you *must* either add `skopeo` as the primary argument or
+    /// indirectly.  However, all other command line options including
+    /// `experimental-image-proxy` will be injected by this library.
+    pub cmd: Option<Command>,
```

Comment:
Should this have `skopeo` in the name to be clearer? E.g. `skopeo_cmd`.

Reply:
Sure, done.